skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -178,7 +177,7 @@ def _nl_means_denoising_2d(image, int s=7, int d=13, float h=0.1):
     cdef int row_start, row_end, col_start, col_end
     cdef int row_start_i, row_end_i, col_start_j, col_end_j
     cdef IMGDTYPE [::1] new_values = np.zeros(n_ch).astype(np.float32)
-    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(util.pad(image,
+    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(np.pad(image,
```

Comment:
Again, thinking of the long game, where crop gets added to numpy, we would want to avoid e.g. tutorials proliferating online using one or the other, instead of the canonical numpy version. So, I'm happy to have it there for legacy purposes, but eventually I wouldn't want to replicate functionality. Our other applications such as Gaussian filters do actually provide useful defaults beyond what's available in ndimage.